lib/core/router/app_router.dart

```diff
@@ -7,7 +7,7 @@ class AppRouter {
   static final GoRouter _goRouter = GoRouter(
       navigatorKey: _rootNavigatorKey,
       debugLogDiagnostics: true,
-      initialLocation: AppPage.splash.toPath,
+      initialLocation: AppPage.onBoarding.toPath,
```

Comment:
The initial location should still be the splash and you will add a timer in the splash screen to the onboarding.